packages/config/src/__tests__/getConfig-test.ts

```diff
@@ -136,7 +138,7 @@ describe(getDynamicConfig, () => {
   it(`throws a useful error for dynamic configs with a syntax error`, () => {
     const paths = getConfigFilePaths('syntax-error');
     expect(() => getDynamicConfig(paths.dynamicConfigPath, mockConfigContext)).toThrowError(
-      'Unexpected token (3:4)'
+      /Error (?:.*) \(5:7\)/
```

Comment:
Don't need the capture group
```suggestion
      /Error .* \(5:7\)/
```